pkg/users/accounts-list.js

```diff
@@ -38,11 +39,51 @@ import { SearchIcon } from '@patternfly/react-icons';
 import { SortByDirection } from "@patternfly/react-table";
 import { account_create_dialog } from "./account-create-dialog.js";
 import { delete_account_dialog } from "./delete-account-dialog.js";
+import { delete_group_dialog } from "./delete-group-dialog.js";
 import { lockAccountDialog } from "./lock-account-dialog.js";
 import { logoutAccountDialog } from "./logout-account-dialog.js";
 
 const _ = cockpit.gettext;
 
+const admins = ['sudo', 'root', 'wheel'];
+
+const sortGroups = groups => {
+    return groups.sort((a, b) => {
+        if (a.isAdmin)
+            return -1;
+        if (b.isAdmin)
+            return 1;
+        if (a.members === b.members)
+            return a.name.localeCompare(b.name);
+        else
+            return b.members - a.members;
+    });
+};
+
+const GroupActions = ({ group, accounts }) => {
+    const [isKebabOpen, setKebabOpen] = useState(false);
+
+    if (!superuser.allowed)
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17980-20221203-084140-31427c73-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)